daemon/cmd/daemon_main.go

```diff
@@ -782,9 +782,6 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.Int(option.NeighMapEntriesGlobalName, option.NATMapEntriesGlobalDefault, "Maximum number of entries for the global BPF neighbor table")
 	option.BindEnv(vp, option.NeighMapEntriesGlobalName)
 
-	flags.Int(option.PolicyMapEntriesName, policymap.MaxEntries, "Maximum number of entries in endpoint policy map (per endpoint)")
```

Comment:
We should document this flag removal in the upgrade guide.

Reply:
This flag is not removed, but merely moved to the new Cell. There should be no upgrade impact because of this :-)